ow_regolith/models/sphere_2cm/model.sdf

```diff
@@ -8,7 +8,7 @@
 
       <pose>0 0 0.01 0 0 0</pose>
       <inertial>
-        <mass>0.3769911184307752</mass>
+        <mass>0.031</mass>
```

Comment:
This is a huge mass change. You should add a comment describing how this mass was decided.

Reply:
Yep! I do not recall what I based the previous mass on, but I'm guessing I made a grams -> kilograms conversion error.